WcaOnRails/app/models/regulation.rb

```diff
@@ -0,0 +1,16 @@
+class Regulation < SimpleDelegator
+  begin
+    REGULATIONS = JSON.parse(File.read("app/views/search_results/wca-regulations.json"))
```

Comment:
Since the file is here in git, you can and should test this. Once you have the test, there is no need for the rescue.

Also, you might as well `freeze` the result.